tests/resources/examples_erroneous/note.jsonl

```diff
@@ -0,0 +1,4 @@
+{"note_id":1001,"person_id":1,"note_date":"2020-10-10","note_datetime":"2020-10-10 01:01:01","note_type_concept_id":3522232,"note_class_concept_id":0,"note_title":"Administrative note","note_text":"This note contains a few characters like ,/\\.[]()*&^$%@&#%","encoding_concept_id":1147732,"language_concept_id":4180186,"provider_id":1234,"visit_occurrence_id":123456,"visit_detail_id":1234567,"note_source_value":5}
+{"note_id":1002,"person_id":2,"note_date":"2020-10-10","note_datetime":"2020-10-10 01:01:01","note_type_concept_id":3522232,"note_class_concept_id":0,"note_title":"Administrative note","note_text":"This is a different note that contains a few characters like ,/\\.[]()*&^$%@&#%","encoding_concept_id":1147732,"language_concept_id":4180186,"provider_id":1234,"visit_occurrence_id":23456,"visit_detail_id":234567,"note_source_value":6}
+{"note_id":1003,"person_id":2,"note_date":"2020-10-10","note_datetime":"2020-10-10 01:01:01","note_type_concept_id":3522232,"note_class_concept_id":0,"note_title":"Administrative note","note_text":"This is also a different note that contains a few characters like ,/\\.[]()*&^$%@&#%","encoding_concept_id":1147732,"language_concept_id":4180186,"provider_id":1234,"visit_occurrence_id":23456,"visit_detail_id":null,"note_source_value":7}
+{"note_id":1003,"person_id":2,"note_date":"2020-10-10","note_datetime":"2020-10-10 01:01:01","favorite_note_id":3522232,"note_class_concept_id":0,"note_title":"Administrative note","note_text":"This is also a different note that contains a few characters like ,/\\.[]()*&^$%@&#%","encoding_concept_id":1147732,"language_concept_id":4180186,"provider_id":1234,"visit_occurrence_id":23456,"visit_detail_id":null,"note_source_value":7}
```

Comment:
Pretty quick! I duplicated this to ~500 MB and it took ~5 seconds to run. Nice work!